src/main.ts

```diff
@@ -12,33 +12,35 @@ loadLocal();
 
 const scriptRegex = /generate-([a-zA-Z\\-]+)\.ts/;
 
-const excludedScripts = ['pretty-manifest'];
+const excludedScripts = ['pretty-manifest', 'font-glyph-enums', 'symbols'];
```

Comment:
`generate-symbols` actually has def dependencies, since it provides def hashes for localization purposes.